src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/SQLFunctions.java

```diff
@@ -183,6 +192,19 @@ public static boolean isFunctionTranslatedToScript(String function) {
             case "second_of_minute":
                 functionStr = dateFunctionTemplate("secondOfMinute", (SQLExpr) paramers.get(0).value);
                 break;
+            case "timestamp":
+                functionStr = timestamp((SQLExpr) paramers.get(0).value);
+                break;
+            case "maketime":
+                functionStr = maketime((SQLExpr) paramers.get(0).value, (SQLExpr) paramers.get(1).value,
+                        (SQLExpr) paramers.get(2).value);
+                break;
+            case "now":
+                functionStr = now();
+                break;
+            case "curdate":
```

Comment:
I think this should be either currentdate or current_date; none of the other names seem to be truncated.

Reply:
This was referred from MySQL documentation, and also we need to support BI integration.